cylc/flow/scripts/trigger.py

```diff
@@ -29,10 +29,28 @@
   # start a new "flow" by triggering 1234/foo
   $ cylc trigger --reflow my_flow//1234/foo
 
-Note: waiting tasks that are queue-limited will be queued if triggered, to
-submit as normal when released by the queue; queued tasks will submit
-immediately if triggered, even if that violates the queue limit (so you may
-need to trigger a queue-limited task twice to get it to submit immediately).
+Manually triggering a task queues it to run regardless of satisfaction of its
+prerequisites. (This refers to Cylc internal queues).
+
+Manually triggering a queued task causes it to submit immediately, regardless
+of the queue limit.
+
+Manually triggering an active task has no effect, because multiple concurrent
+instances of the same task job aren't allowed. Active tasks are in the
+submitted or running states.
+
+Manually triggering an incomplete task queues it to run again and continue its
+assigned flow. Incomplete tasks finished without completing expected outputs.
```

Comment:
```suggestion
Manually triggering an incomplete task (a task that finished without completing
expected outputs) queues it to run again and continue its assigned flow.
```

Reply:
I deliberately changed it to this way around to get the important information first in one concise sentence (*Triggering an xxx task does this.*), followed by some additional information for those who need a reminder of what an "incomplete task" actually is, etc.

Reply:
Then perhaps
```suggestion
Manually triggering an incomplete task queues it to run again and continue its
assigned flow (incomplete tasks are those which finished without completing
expected outputs).
```

Reply:
Fine either way.